i18n/en-US/core/stats.json

```diff
@@ -0,0 +1,38 @@
+{
+    "DESCRIPTION": "Get TypicalBot's current statistics.",
+    "USAGE": "stats",
+    "TEXT": "$t(core/stats:TEXT_1)\n$t(core/stats:TEXT_2)\n$t(core/stats:TEXT_3)\n$t(core/stats:TEXT_4)\n$t(core/stats:TEXT_5)\n$t(core/stats:TEXT_6)\n$t(core/stats:TEXT_7)\n$t(core/stats:TEXT_8)\n$t(core/stats:TEXT_9)\n$t(core/stats:TEXT_10)\n$t(core/stats:TEXT_11)\n$t(core/stats:TEXT_12)\n\n$t(core/stats:TEXT_13)\n$t(core/stats:TEXT_14)\n$t(core/stats:TEXT_15)\n$t(core/stats:TEXT_16)\n$t(core/stats:TEXT_17)\n$t(core/stats:TEXT_18)\n$t(core/stats:TEXT_19)\n$t(core/stats:TEXT_20)\n$t(core/stats:TEXT_21)",
+    "TEXT_1": "**__TypicalBot's Statistics:__**",
+    "TEXT_2": "```autohotkey",
+    "TEXT_3": "=> Uptime: {{uptime}}",
+    "TEXT_4": "=> Servers: {{guilds}} ({{count}} Shards})",
+    "TEXT_5": "=> Voice Connections: {{voiceConnections}}",
+    "TEXT_6": "=> Channels: {{channels}}",
+    "TEXT_7": "=> Users: {{users}}",
+    "TEXT_8": "=> CPU: {{cpu}}%",
+    "TEXT_9": "=> RAM (Used): {{usedRAM}}MB",
+    "TEXT_10": "=> RAM (Total): {{totalRAM}}MB",
+    "TEXT_11": "=> Library: discord.js",
+    "TEXT_12": "=> Created By: HyperCoder#2975 & nsylke#4490",
+    "TEXT_13": "    This Shard:",
+    "TEXT_14": "=> Cluster: {{clusterName}} (Shards: {{clusterShards}})",
+    "TEXT_15": "=> Servers: {{currentGuilds}}",
+    "TEXT_16": "=> Voice Connections: {{currentVoiceConnections}}",
+    "TEXT_17": "=> Channels: {{currentChannels}}",
+    "TEXT_18": "=> Users: {{currentUsers}}",
+    "TEXT_19": "=> RAM (Used): {{ram}}MB",
+    "TEXT_20": "=> RAM (Total): {{ramTotal}}MB",
+    "TEXT_21": "```",
+    "TYPICAL_STATS": "TypicalBot Statistics",
+    "UPTIME": "» Uptime",
+    "SERVERS": "» Servers",
+    "SERVERS_VALUE": "{{guilds}} ({{count}} Shards})",
+    "VOICE": "» Voice Connections",
+    "CHANNELS": "» Channels",
+    "USERS": "» Users",
+    "CPU": "» CPU Usage",
+    "RAM": "» RAM (Used)",
+    "RAM_TOTAL": "» RAM (Total)",
+    "LIBRARY": "» Library",
+    "CREATED_BY": "» Created By"
```

Comment:
After uploading the strings to Crowdin, these special characters will not render properly and/or become other characters. Can we use unicode instead? The unicode for `»` is `\u00bb`.